README.md

```diff
@@ -647,10 +647,26 @@ python3 -m ethstaker_deposit [OPTIONS] COMMAND [ARGS]
 
 ### Building Binaries
 **Developers Only**
-##### Mac M1 Binaries
 
-👋This is not the section you are looking for.👋
-If you are trying to **build the binary** on macos with an M1 Mac and you are using pyenv to manage your python version. You'll probably need to reinstall a given python version using:
-```
-env PYTHON_CONFIGURE_OPTS="--enable-framework" pyenv install 3.10.3
+ethstaker-deposit uses `pyinstaller` to create binaries. The requirements are in `build_configs`. Look at `.circleci/config.yml` to see it in action.
+
+For example Linux, in your Python virtual environment:
+```sh
+export BUILD_FILE_NAME=ethstaker_deposit-cli-dev-linux
+pip install -r ./build_configs/linux/requirements.txt
+pyinstaller --distpath ./${BUILD_FILE_NAME} ./build_configs/linux/build.spec
 ```
+
+## Canonical Deposit Contract and Launchpad
```

Comment:
Can this section be added to `docs/src/landing.md` as well?

Reply:
Absolutely, done. Let me know whether it's OK where it is.

Reply:
Thanks! Looks good.